src/main/java/org/opentripplanner/middleware/auth/Auth0Users.java

```diff
@@ -244,21 +242,19 @@ private static String getAuth0Url() {
     /**
      * Get an Auth0 oauth token for use in mocking user requests by using the Auth0 'Call Your API Using Resource Owner
      * Password Flow' approach. Auth0 setup can be reviewed here: https://auth0.com/docs/flows/call-your-api-using-resource-owner-password-flow.
-     * If the user is successfully validated by Auth0 a bearer access token is returned, which is extracted and returned
-     * to the caller. In all other cases, null is returned.
+     * If the user is successfully validated by Auth0 a complete token is returned. In all other cases, null is returned.
      */
-    public static String getAuth0Token(String username, String password) throws JsonProcessingException {
+    public static TokenHolder getCompleteAuth0Token(String username, String password) {
         if (Auth0Connection.isAuthDisabled()) return null;
         String body = String.format(
             "grant_type=password&username=%s&password=%s&audience=%s&scope=&client_id=%s&client_secret=%s",
             username,
             password,
-            "https://otp-middleware", // must match an API identifier
+            DEFAULT_AUDIENCE, // must match an API identifier
             AUTH0_CLIENT_ID, // Auth0 application client ID
             AUTH0_CLIENT_SECRET // Auth0 application client secret
```

Comment:
These variables are only for e2e testing per README, so this call will not work outside of the e2e environment.

Reply:
Have updated to use AUTH0_API_CLIENT and AUTH0_API_SECRET instead.